### ISSUE TITLE
Replace existing SPDX comment if it exists

### ISSUE THREAD

**Issue by lassik** [condy0919/spdx.el#6]
Would you accept a feature to find existing `Copyright`/`SPDX-License-Identifier` lines at the top of the file and edit (i.e. replace) them if found?

**Comment by condy0919** [comment]
It seems like a valuable feature. @riscy has proposed the same in https://github.com/melpa/melpa/pull/7250

**Comment by lassik** [comment]
`parse-partial-sexp` and `syntax-ppss` can be used to find all the comments in a buffer.

**Comment by condy0919** [comment]
It's sexp only?

**Comment by lassik** [comment]
No, any syntax. The function names say `sexp` for historical reasons.
